wagtailnews/views/editor.py

```diff
@@ -76,7 +77,14 @@ def create(request, pk):
     else:
         form = EditForm(instance=newsitem)
 
-    edit_handler = edit_handler.bind_to_instance(instance=newsitem, form=form)
+    if WAGTAIL_VERSION == '2.1':
```

Comment:
This will break when Wagtail 2.2 is released. The wagtail version is exposed as [a tuple at `wagtail.VERSION`](https://github.com/wagtail/wagtail/blob/master/wagtail/__init__.py), which you can use to make comparisons against. Try something like this instead:
```python
import wagtail
if wagtail.VERSION >= (2, 1):
    # ...
else:
    # ...
```